lib/git_reflow/git_server/pull_request.rb

```diff
@@ -0,0 +1,97 @@
+module GitReflow
+  module GitServer
+    class PullRequest
+      attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
+
+      def initialize(attributes)
+        raise "PullRequest#initialize must be implemented"
+      end
+
+      def commit_author
+        raise "#{self.class.to_s}#commit_author method must be implemented"
+      end
+
+      def comments
+        raise "#{self.class.to_s}#comments method must be implemented"
+      end
+
+      def has_comments?
+        comments.count > 0
+      end
+
+      def last_comment
+        raise "#{self.class.to_s}#last_comment_for method must be implemented"
+      end
+
+      def reviewers
+        raise "#{self.class.to_s}#reviewers method must be implemented"
+      end
+
+      def approvals
+        raise "#{self.class.to_s}#approvals method must be implemented"
+      end
+
+      def reviewers_pending_response
+        reviewers - approvals
+      end
+
+      def good_to_merge?(force: false)
```

Comment:
I like how you've abstracted this so it isn't implemented elsewhere. Very clean!